Web/Admin.UI/src/app/components/validation-config/validation-categories/validation-rule-delete-dialog/validation-rule-delete-dialog.component.html

```
@@ -0,0 +1,18 @@
+<h1 mat-dialog-title>Confirm Delete</h1>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)